app/views/user_posts/index.html.erb

```diff
@@ -1 +1,37 @@
-<h1>All posts by a given user</h1>
+<main>
+  <section>
+    <div class="card">
+      <div class="user-image border"></div>
+      <div class="user-data border">
+        <h1><%= @user.name %></h1>
+        <p class="number-posts">Number of posts: <%= @user.posts_counter %></p>
+      </div>
+    </div>
+  </section>
+  <section>
+    <ul>
+      <% @posts.each do |post| %>
+        <%= link_to user_user_post_path(user_id: post.author_id, id: post.id) do %>
+          <li class="card hover flexColumn">
+            <div class="card flexColumn">
+              <h2><%= post.title %></h2>
+              <p><%= post.text %></p>
+              <div class="right">
+                <span>Comments: <%= post.comments_counter %></span>
+                <span>Likes: <%= post.likes_counter %> </span>
+              </div>
+            </div>
+            <div class="commets-container">
+              <h4>Comments:</h4>
+              <ul>
+                <% post.recent_comments.each do |comment| %>
+                  <p><%= comment.author.name %>: <%= comment.text %></p>
+                <% end %>
+              </ul>
+            </div>
+          </li>
+        <% end %>
+      <% end %>
+    </ul>
+  </section>
+</main>
```

Comment:
- [ ] Please 🙏🏽 add the pagination button at the ed of this page to be similar to the wireframe 👇🏽 

| Your design | wireframe |
|---|---|
| ![image](https://github.com/MarcoDDM/blog-app/assets/58642949/b37333b4-1b1f-4da8-9deb-53a7d8ea0331) | ![image](https://github.com/MarcoDDM/blog-app/assets/58642949/77a1ed4c-29e7-429f-98f7-5ca3ae34201c) |